addons/point_of_sale/static/src/js/models.js

```diff
@@ -2214,12 +2233,15 @@ exports.Order = Backbone.Model.extend({
         this.paymentlines.each(_.bind( function(item) {
             return paymentLines.push([0, 0, item.export_as_JSON()]);
         }, this));
+        if(!(this.is_paid_with_cash() && this.get_total_paid() > 0)) {
```

Comment:
why those changes are needed?

Reply:
This is the result of merging the following task in this task:
https://github.com/odoo/odoo/pull/32934